### ISSUE TITLE
Question about inference

### ISSUE THREAD

**Issue by cyysc1998** [opendatalab/FakeVLM#12]
Thanks for sharing the excellent work! I trained with the parameters in the code and tested using my own test code. The training set fits well, but the test set shows an accuracy of only 75%. Below is my test code. Could you take a look to see where the problem might be? Thank you.
```python
# 1. convert the key of model weight to hf format with https://github.com/huggingface/transformers/blob/main/src/transformers/models/llava/convert_llava_weights_to_hf.py#L52

# 2 load model from ckpt, but param name mismatched
model_id = 'path/to/checkpoint'
model = LlavaForConditionalGeneration.from_pretrained(
    model_id, 
    torch_dtype=torch.float16, 
    low_cpu_mem_usage=True, 
).to(0)

# 3 inference with this template
conversation = [
    {
      "role": "system",
      "content": [
          {"type": "text", "text": "A chat between a curious user and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions."}
      ]
    },
    {
      "role": "user",
      "content": [
          {"type": "text", "text": "Does the image looks real/fake?"},
          {"type": "image"},
        ],
    },
]
prompt = processor.apply_chat_template(conversation, add_generation_prompt=True)
inputs = processor(images=raw_image, text=prompt, return_tensors='pt').to(0, torch.float16)
output = model.generate(**inputs, max_new_tokens=200, do_sample=False)
```

**Comment by lingcco** [comment]
Hello, you can refer to our evaluation code. Thank you for your waiting and sorry for our late reply.
